src/Sarif/Writers/PrereleaseCompatibilityTransformer.cs

```diff
@@ -243,6 +246,36 @@ private static bool ConvertRunRedactionTokenToArray(JObject run)
             return false;
         }
 
+        private static bool RenameSuppressionKindValues(JObject run)
+        {
+            string suppressionsPathToUpdate = "results[].suppressions[]";
+
+            bool actionOnLeaf(JObject suppression)
+            {
+                if (suppression["kind"] is JValue kind && kind.Value is string)
+                {
+                    if (kind.Value as string == "suppressedInSource" )
```

Comment:
>kind [](start = 24, length = 4)

you know these are strings already, so just use a cast. also you could capture inReview here and then create a new state object to hold it. we don't do that because it's just not worth the cost though, yes? i.e., no one is populating 'inreview'. i think that's fair

Reply:
convert these to casts and the go right ahead and merge!

---
In reply to: [277075058](https://github.com/Microsoft/sarif-sdk/pull/1442#discussion_r277075058) [](ancestors = 277075058)

Reply:
thanks! based on the other comment, i am going to wait on this to confirm if we actually need to fill the state value here. will work on the cast issue :)

---
In reply to: [277075132](https://github.com/Microsoft/sarif-sdk/pull/1442#discussion_r277075132) [](ancestors = 277075132,277075058)